plugins/woocommerce-admin/client/payments-welcome/apms.tsx

```diff
@@ -0,0 +1,137 @@
+/**
+ * External dependencies
+ */
+import { List } from '@woocommerce/components';
+import { download, Icon } from '@wordpress/icons';
+
+import {
+	Card,
+	CardBody,
+	CardFooter,
+	CardHeader,
+	ExternalLink,
+	Notice,
+	ToggleControl,
+} from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import strings from './strings';
+import { WC_ASSET_URL } from '~/utils/admin-settings';
+
+export interface Apm {
+	id: string;
+	title: string;
+	icon: string;
+	description: string;
+	link: string;
+	extension: string;
+}
+
+export const apms: Apm[] = [
+	{
+		id: 'paypal',
+		title: strings.apms.paypal.title,
+		icon: `${ WC_ASSET_URL }images/payment_methods/72x72/paypal.png`,
+		description: strings.apms.paypal.description,
+		link: 'https://woocommerce.com/products/woocommerce-paypal-payments/',
+		extension: 'woocommerce-paypal-payments',
+	},
+	{
+		id: 'amazonpay',
+		title: strings.apms.amazonpay.title,
+		icon: `${ WC_ASSET_URL }images/payment_methods/72x72/amazonpay.png`,
+		description: strings.apms.amazonpay.description,
+		link: 'https://woocommerce.com/products/pay-with-amazon/',
+		extension: 'woocommerce-gateway-amazon-payments-advanced',
+	},
+	{
+		id: 'klarna',
+		title: strings.apms.klarna.title,
+		icon: `${ WC_ASSET_URL }images/payment_methods/72x72/klarna.png`,
+		description: strings.apms.klarna.description,
+		link: 'https://woocommerce.com/products/klarna-payments/',
+		extension: 'klarna-payments-for-woocommerce',
+	},
+];
+
+interface ApmsProps {
```

Comment:
_Nitpick_: We can rename it to `ApmListProps` to match new component name.

Reply:
Fixed in https://github.com/woocommerce/woocommerce/pull/34581/commits/579b1d63f1d8fdb76ca4449af6d23d873e438f09!